backend/migration.js

```diff
@@ -36,26 +39,31 @@ function runMigration(loggerFactory, dbModule) {
     function initUmzug() {
         const { sequelize } = db;
         umzug = new Umzug({
-            storage: 'sequelize',
-            storageOptions: {
-                sequelize
-            },
+            storage: new SequelizeStorage({ sequelize }),
 
             // see: https://github.com/sequelize/umzug/issues/17
```

Comment:
Is this comment still valid?

Reply:
I would say it was useless even before the change - removed